libraries/AC_Avoidance/AP_OADatabase.cpp

```diff
@@ -329,6 +446,25 @@ void AP_OADatabase::database_item_refresh(const uint16_t index, const uint32_t t
 
 void AP_OADatabase::database_items_remove_all_expired()
 {
+    // we expect tracked objects to be updated at atleast 1hz
+    if (_tracked_obs_enable) {
+        for (uint16_t i = 0; i < _database_tracked.count; i++) {
+            if (_database_tracked.items[i].id == UINT16_MAX) {
+                //unassigned
```

Comment:
maybe change this comment to, "// skip items with no id" or "// skip items with unassigned id"